src/Console/Helper/DumpInfo.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Smile\GdprDump\Console\Helper;
+
+use Smile\GdprDump\Database\Metadata\MetadataInterface;
+use Smile\GdprDump\Dumper\Config\DumperConfig;
+use Smile\GdprDump\Dumper\Event\DumpEvent;
+use Smile\GdprDump\Dumper\Event\DumpFinishedEvent;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class DumpInfo
+{
+    private OutputInterface $output;
+    private ProgressBar $progressBar;
+
+    public function __construct(private EventDispatcherInterface $eventDispatcher)
+    {
+    }
+
+    public function setOutput(OutputInterface $output): void
+    {
+        if ($output instanceof ConsoleOutputInterface) {
+            $output = $output->getErrorOutput();
+        }
+
+        $this->output = $output;
+        $this->progressBar = new ProgressBar($output);
+
+        $this->eventDispatcher->addListener(DumpEvent::class, $this->getDumpStartedListener());
+        $this->eventDispatcher->addListener(DumpFinishedEvent::class, $this->getDumpFinishedListener());
+    }
+
+    /**
+     * Get the listener that will be triggered when a dump is started.
+     */
+    private function getDumpStartedListener(): callable
+    {
+        return function (DumpEvent $event): void {
+            $config = $event->getConfig();
+            $database = $event->getDatabase();
+
+            // DSN
+            $this->displaySection('Database settings');
+            $this->displaySectionItem('Dsn', $database->getDriver()->getDsn());
+            $this->displaySectionItem(
+                'With password',
```

Comment:
nit: I am no english native, but I feel this would sound more natural:

```suggestion
                'Using password',
```